lib/active_merchant/billing/gateways/airwallex.rb

```diff
@@ -244,7 +244,12 @@ def add_order(post, options)
         shipping[:last_name] = last_name if last_name
         shipping[:phone_number] = shipping_address[:phone_number] if shipping_address[:phone_number]
         shipping[:address] = physical_address
-        post[:order] = { shipping: }
+
```

Comment:
I think we may need to add `options[:products]` to the guard clause at the top of the method, and skip the shipping address logic if no shipping address is provided